Archive/MiuLanguage/DecisionNec.lean

```diff
@@ -154,9 +154,7 @@ theorem goodm_of_rule3 (as bs : Miustr) (h₁ : Derivable (as ++ ↑[I, I, I] ++
     exact mhead
   · contrapose! nmtail
     rcases exists_cons_of_ne_nil k with ⟨x, xs, rfl⟩
-    -- Porting note: `simp_rw [cons_append]` didn't work
-    rw [cons_append] at nmtail; rw [cons_append, cons_append]
-    dsimp only [tail] at nmtail ⊢
+    simp_rw [cons_append _] at nmtail ⊢
```

Comment:
I assume this doesn't work?
```suggestion
    simp_rw [cons_append] at nmtail ⊢
```
If so, maybe replace the proof but keep the porting note, as this would be a genuine regression

Reply:
Right, the underscore is necessary. There were a few places like this.

Reply:
Then I would suggest keeping the porting notes. This is worth investigating

Reply:
It seems the underscore is no longer needed. I'm not sure when this got fixed.